### ISSUE TITLE
Can't install Canvas (Windows)

### ISSUE THREAD

**Issue by Senen42** [Automattic/node-canvas#1025]
Hi.

I followed the wiki to install node-canvas and it doesn't work. Not used to npm so I have no idea what I'm doing but I tried both of the method (the one using Choco and the manual way), both seems to lead to the same error when I use $npm install canvas. Everything seems to install properly up to that command. For the manual way, I installed everything (Node.js, GTK and libjpeg) in 64bit so that shouldn't be the problem.

Here's the log: https://pastebin.com/2fHJRhae

**Comment by Senen42** [comment]
Ok found the solution. I had to browse through A LOT of old threads here to find it while several other people seemed to had the same problem and didn't get to solve it, so I hope it'll be useful to some: 

I tried to install node-canvas as part of another installation process (Discord Spoiler Bot) so I was just following several installation manuals having no idea what I was installing and how it's supposed to be used. Turns out, by default I was installing everyting in C:\Windows\System32, which didn't seem that surprising for me. Except that apparently the "npm install canvas" is not supposed to be used there, but inside the folder of your project. I made a folder C:\DiscordSpoilerBot, went in there (if you're not used to command lines, in that example you'd want to use $ cd ..\..\DiscordSpoilerBot) and then tried to install canvas, and it worked.
